### PR TITLE
fix passing extra option to docker

### PR DESCRIPTION
**Pull request by antonkol**

### CODE REVIEW

File: scripts/iscan [hunk 1]
```diff
@@ -254,15 +258,15 @@ cmd_scan() {
         $DOCKER run --rm \
             --mount readonly,type=bind,source="$volume",destination=/vol \
             --env STEM="$name" \
-            $env_extra \
             $image \
-            --for $opt_for /vol
+            --for $opt_for \
+            "${extra_params[@]}" /vol
```
Comment (vvv) [inline scripts/iscan:263]:
```suggestion
            --for $opt_for \
            ${ISCAN_EXTRA_PARAMS:-} \
            /vol
```
Can't we do this instead?

**Review by vvv — COMMENTED** [review]
1. Sorry for breaking your code in the first place.
2. I don't see the trickery with `extra_params` array is needed.
3. Feel free to sacrifice the comment at lines 245–246 or move it closer to the section with docker run-s.

File: scripts/iscan (hunk 1 above)
Comment (antonkol) [reply scripts/iscan:263]: yep, will fo

Comment (antonkol) [reply scripts/iscan]: done

**Review by vvv — APPROVED** [review]

**Pull request merged**